extensions/amp-story/1.0/amp-story-interactive.js

```diff
@@ -381,6 +398,19 @@ export class AmpStoryInteractive extends AMP.BaseElement {
       true /** callToInitialize */
     );
 
+    // Check if the component page is active, and add class.
+    this.storeService_.subscribe(
+      StateProperty.CURRENT_PAGE_ID,
+      (currPageId) => {
+        if (currPageId === this.getPageId_()) {
```

Comment:
You'd also need to remove the active class when navigating away

Reply:
👀

Reply:
Sounds good for removing the active class. Meant anything else with the 👀?

Reply:
Nope I wanted to make sure you saw it as I sent it while you were pushing commits.

Can you remove the class in the `mutateElement` as well? You can use `classList.toggle(className, bool)`